src/getAverage.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ * Implement a custom array method getAverage:
+ *
+ * Method works in different ways:
```

Comment:
Function calculates the average value based on passed options. 
1. if options are not passed, returns average value of all items in the array
2. if option.propertyName is passed, returns average value of `propertyName` property of all items in the array.
3. if option.accumulator = function(item, index, array) is passed, returns average value of accumulator result applied to all items in the array.

Reply:
will change passed argument to `options` with this description

Reply:
@roman-mate @mgrinko should we pass `array` argument to the accumulator function? We don't have tests to check it.

Reply:
And what is the behavior of `array` argument here?
```
users.calculateAverage({
  accumulator: (user, index, array) => user.age + (index * 100)) === 130
}
```
We're not able to change the array option here like in other array methods. Should we just check `array === users` in this case?

Reply:
yeah, since we have item and index, array feels useless. maybe let's not pass it.

Reply:
@roman-mate Good description, thanks!

Reply:
I added `array` as a third argument as it is natural for all the iteration methods
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#Parameters

Reply:
It is very easy to test

Reply:
good point. then let's keep it.